pages/configuration/replication.mdx

```diff
@@ -15,10 +15,10 @@ Memgraph 2.9 introduced a new configuration flag
 If you want instances to remember their role and configuration in a replication
 cluster upon restart, the `--replication-restore-state-on-startup` needs to be
 set to `true` when first initializing the instances and remain `true` throughout
-the instances' lifetime. 
+the instances' lifetime.
```

Comment:
@andrejtonev this resolves this PR: https://github.com/memgraph/memgraph/pull/1707, right?

Reply:
I think we need to explicitly tell the user they need to re-register on MAIN a replica if running without this flag.
Right?

Reply:
> I think we need to explicitly tell the user they need to re-register on MAIN a replica if running without this flag. Right?

@antoniofilipovic and  @andrejtonev are any additional documentation updates need when it comes to this or can I merge the docs?

Reply:
Feel free to merge, I will open new PR for this

Reply:
Done in new PR